doc/api/crypto.md

```diff
@@ -522,6 +522,12 @@ provided, `otherPublicKey` is expected to be a [`Buffer`][],
 If `outputEncoding` is given a string is returned; otherwise, a
 [`Buffer`][] is returned.
 
+*Note*:
+
+* The `server.computeSecret` will throw an 
```

Comment:
Shouldn't this be `diffieHellman.computeSecret()`?

Also we can drop 'The' at the beginning.

Reply:
my mistake :) 
already corrected 👍

Reply:
my mistake :)
already corrected 👍  thanks!